client/state/utils/actions.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( prefix ) {
```

Comment:
I'm curious why we specify `prefix` here. Is is simply a helper to make it easy to embed the counter in a string? The store clearly doesn't care about the prefix since it's never passed, meaning that `prefix` doesn't appear to have any meaning with regards to the unique id.

Is it worth adding here? Is it here just to mimic **lodash**?